crates/wdk-build/src/lib.rs

```diff
@@ -187,6 +187,10 @@ pub enum ConfigError {
     #[error(transparent)]
     CargoMetadataError(#[from] cargo_metadata::Error),
 
+    /// Error returned when no wdk-build crate is detected
+    #[error("no wdk-build crate was detected in this workspace")]
```

Comment:
```suggestion
    #[error("no wdk-build package was detected in this workspace")]
```
Makes this error message consistent with the error message on the `MultipleWdkBuildCratesDetected` error

Reply:
Thanks for the suggestion. I've updated the error message to match the style of the `MultipleWdkBuildCratesDetected` error.